File: 2025-1/sesion-de-clase-04/PUCP/Edu/Pe/Inf30/Sesion04/Bodega.cs
using System;
using PUCP.Edu.Pe.Inf30.Sesion04.Entidades;

namespace PUCP.Edu.Pe.Inf30.Sesion04 {
	public class Bodega {
		public static void Main(string[] args) {
			Producto p1 = new Producto();
			p1.Codigo = 1000;
			p1.Nombre = "Arroz";
			p1.Stock = 20;
			p1.Precio = 5.00;
			
			p1.imprimir();
		}
	}
}

File: 2025-1/sesion-de-clase-04/pe/edu/pucp/inf30/sesion04/Bodega.java
package pe.edu.pucp.inf30.sesion04;

import pe.edu.pucp.inf30.sesion04.entidades.*;

public class Bodega {
	public static void main(String[] args) {
		Producto p1 = new Producto();
		p1.setCodigo(1000);
		p1.setNombre("Arroz");
		p1.setStock(20);
		p1.setPrecio(5.00);
		
		p1.imprimir();
	}
}